api/routes/routes.js

```diff
@@ -57,4 +58,7 @@ module.exports = function (app) {
   app
     .route("/update_to_collection")
     .put(collectionController.updateKratooToCollection);
+
+  // Content related enpoint
+  app.route("/user_content").get(contentController.userContent);
```

Comment:
I already write the endpoint for getting kratoo by kratoo id.

`app.route("/kratoo/:kratooId").get(pinController.getKratooByKratooId);`